app/Http/Controllers/Bible/LibraryController.php

```diff
@@ -497,7 +497,7 @@ public function volume()
                     $query->where('bible_filesets.updated_at', '>', $updated);
                 })
                 ->when($version_code, function ($query) use ($version_code) {
-                    $query->whereRaw('SUBSTRING(bibles.id,4) = ?', [$version_code]);
+                    $query->where('bible_filesets.id', 'LIKE', '%' . $version_code .'%')->get();
```

Comment:
It has been replaced by the query that is used for the `version` action. For now on, it will try to do matching between `bible_filesets.id` and `$version_code` request parameter